frontend/src/pages/RunDetails.tsx

```diff
@@ -853,19 +864,36 @@ class RunDetails extends Page<RunDetailsInternalProps, RunDetailsState> {
   private async _loadSidePaneTab(tab: SidePaneTab): Promise<void> {
     const workflow = this.state.workflow;
     const selectedNodeDetails = this.state.selectedNodeDetails;
+
+    let sidepanelBannerMode = 'warning' as Mode;
```

Comment:
nit: `as` is a force type cast in typescript, recommend more idiomatic usage to specify type.
```
let sidepanelBannerMode: Mode = 'warning';
```